### PR TITLE
Fixes trees being gin in the codex

### PR DESCRIPTION
**Pull request by DominikPanic**
Kek

### CODE REVIEW

File: code/modules/reagents/chemistry_reagents/alcohol.dm [hunk 1]
```diff
@@ -167,7 +167,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	name = "Gin"
 	description = "It's gin. In space. I say, good sir."
 	color = "#664300" // rgb: 102, 67, 0
-	taste_description = "an alcoholic christmas tree"
+	taste_description = "wood"
```
Comment (Ghommie) [inline code/modules/reagents/chemistry_reagents/alcohol.dm]: Just googled up, and it says gin has a pine aroma... wood is too "beavery", is it really necessary?

Comment (DominikPanic) [reply code/modules/reagents/chemistry_reagents/alcohol.dm]: Make a suggestion and i'll merge it, this was just the simplest I could think up

Comment (Ghommie) [reply code/modules/reagents/chemistry_reagents/alcohol.dm]: "pine" would suffice I guess.

Comment (DominikPanic) [reply code/modules/reagents/chemistry_reagents/alcohol.dm]: Feel free to suggest it

Comment (Ghommie) [reply code/modules/reagents/chemistry_reagents/alcohol.dm]:
```suggestion
	taste_description = "pine"
```
Miscommunication at its best.

**Pull request merged**